src/components/DataCollector/index.js

```diff
@@ -49,15 +49,15 @@ const createDataCollector = ({ config }) => {
 
   const createEventHandler = options => {
     const event = createEvent();
-    const { viewStart = false, data, meta } = options;
+    const { viewStart = false, exitLink = false, data, meta } = options;
 
     event.mergeData(data);
     event.mergeMeta(meta);
 
     return lifecycle
-      .onBeforeEvent(event, options, viewStart)
+      .onBeforeEvent(event, options, viewStart, exitLink)
```

Comment:
I wonder it it's time to change this signature to `{ event, options, viewStart, exitLink }`

Reply:
@Aaronius @artur-ciocanu

Reply:
I'm good either way. It's a little odd that `onBeforeDataCollection` would still just be `payload`. These decisions give me nightmares. Make it stop.

Reply:
I think moving to `({...})` now, for all hooks, can set us up for the future when we want to pass more params.

Reply:
Upon further review, I agree with this. ☝️